controllers/featureFlags.js

```diff
@@ -82,9 +84,33 @@ const deleteFeatureFlag = async (req, res) => {
   }
 }
 
+/**
+ * Get featureFlag config.
+ *
+ * This returns a list of feature flags and whether they are enabled or not.
+ * Feature flag being enabled depends on rollout config.
+ */
+const getConfig = async (req, res) => {
+  try {
+    const featureFlags = await featureFlagQuery.fetchFeatureFlags(false)
+    return res.json({
+      data: featureFlags.map((featureFlag) => {
+        return {
```

Comment:
Since we're already using arrow function here and not doing anything before return, can we simply use () => () to return it? I'm talking about omitting the `return` keyword

Reply:
Done. How do you catch such things? 🤔